src/pickley/bstrap.py

```diff
@@ -319,13 +326,13 @@ def main(args=None):
         if sys.version_info[:2] <= (3, 7):
             package_manager = "pip==21.3.1"
 
-        elif bstrap.pickley_version >= "4.3":  # Temporary: continue using pip by default until 4.3+
+        elif pickley_version >= "4.3":  # Temporary: continue using pip by default until 4.3+
```

Comment:
is this comparing strings?  Do you promise to never use two-digit major or minor?

Reply:
This is comparing strings, and only during the 4.2 -> 4.3 transition
After that, the whole thing should be simplified to always default to `uv` unless otherwise explicitly configured